### PR TITLE
fit BottomSheet not to be displayed behind navbar

### LINKED ISSUE #893: The navigation bar covers bottom sheet contents
**Describe the bug**
As the title says, the navigation bar covers bottom sheet contents.

**To Reproduce**
1. Enable the navigation bar, or run on old devices
2. Go to the search screen
3. Tap the 'Day' button

**Expected behavior**
The navigation bar doesn't cover them.

**Screenshots**

(Pixel 2 API 24 Emulator)

### PR DESCRIPTION
**Pull request by ked4ma**
## Issue
- close #893

## Overview (Required)
- When search filter bottomsheet is displayed on older devices, bottomsheet is displayed behind navbar.
- Display bottomsheet to be able to click on these devices.

## Links
- [App Design](https://www.figma.com/file/NcSMs6dMsD88d4wOY0g3rK/DroidKaigi-2022-Conference-App?node-id=0%3A1)

## Screenshot
Device | Before | After
:--: | :--: | :--:
Pixel2 API 24 |   |  
Pixel6 API 33 |   |   

## Movie
Device | Before | After
:--: | :--: | :--:
Pixel2 API 24 | <video src="https://user-images.githubusercontent.com/74723074/194753406-9da5f95a-9802-4168-ab39-0544389cbc4e.mp4" type="video/mp4" /> | <video src="https://user-images.githubusercontent.com/74723074/194753650-8a9b179f-bd22-42ef-8834-fe09ea099a19.mp4" type="video/mp4" />
Pixel6 API 33 | <video src="https://user-images.githubusercontent.com/74723074/194753431-3ab3d5e5-6d88-4758-88f6-3f941bdea098.mp4" type="video/mp4" /> | <video src="https://user-images.githubusercontent.com/74723074/194753440-6a05b303-68ed-4588-b0db-2dd71c7edc8a.mp4" type="video/mp4" />

### CODE REVIEW

**Comment by takahirom** [comment]
👀

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt [hunk 1]
```diff
@@ -141,7 +146,8 @@ fun SearchRoot(
                         selectedCategories = state.filter.selectedCategories,
                         categories = sheetState.categories,
                         onCategoriesSelected = viewModel::onCategoriesSelected,
-                        onDismiss = viewModel::onFilterSheetDismissed
+                        onDismiss = viewModel::onFilterSheetDismissed,
+                        contentPadding = WindowInsets.navigationBars.asPaddingValues(),
```
Comment (takahirom) [inline feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:150]: I have a question 🙋.
Can we use content padding on all sheets? Do we need to use it differently when we have more or less content?

**Comment by takahirom** [comment]
Great videos! Thanks 🙏

**Comment by takahirom** [comment]
But I think it is better than the current implementation! So I'll merge this! Thanks!

File: feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt (hunk 1 above)
Comment (ked4ma) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:150]: This padding will be set to LazyList so if a sheet is composed only with LazyList, I think "yes".
But if we have to use other composables under the LazyList, we need to change here.

Comment (takahirom) [reply feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt:150]: That makes sense! Thanks!

**Pull request merged**